plugins/woocommerce/includes/abstracts/abstract-wc-data.php

```diff
@@ -19,7 +19,7 @@
  *
  * Implemented by classes using the same CRUD(s) pattern.
  *
- * @version  2.6.0
+ * @version  x.x.x
```

Comment:
So in core we don't rename it to `x.x.x`, we add the exact version, i.e the version this PR will be released with.

Also, I think we add @since to preserve the original version.